### PR TITLE
[#IOPID-2331] `FF_UNIQUE_EMAIL_ENFORCEMENT` removal

### PR DESCRIPTION
**Pull request by salvatorelaiso**
#### List of Changes
Remove Feature Flag `FF_UNIQUE_EMAIL_ENFORCEMENT` and its usage from `GetProfile` and `UpdateProfile`.
Linting.

#### Motivation and Context
The Feature Flag `FF_UNIQUE_EMAIL_ENFORCEMENT` is not needed anymore.

#### How Has This Been Tested?
Unit tests.

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by arcogabbo — CHANGES_REQUESTED** [review]
it seems you forgot the variable `FF_UNIQUE_EMAIL_ENFORCEMENT` inside `IConfig` decoder in `config.ts` file

**Comment by salvatorelaiso** [comment]
> it seems you forgot the variable `FF_UNIQUE_EMAIL_ENFORCEMENT` inside `IConfig` decoder in `config.ts` file

Solved in [ed8c8d7](https://github.com/pagopa/io-functions-app/pull/317/commits/ed8c8d7595334f534aca4d47a8009d1b6aa13be1)

**Review by arcogabbo — APPROVED** [review]

**Review by gquadrati — CHANGES_REQUESTED** [review]
LGTM, I just left a small comment

File: utils/config.ts [hunk 1]
```diff
@@ -128,8 +128,6 @@ export const IConfig = t.intersection([
     FF_OPT_IN_EMAIL_ENABLED: t.boolean,
     FF_TEMPLATE_EMAIL: FeatureFlagFromString,
 
-    FF_UNIQUE_EMAIL_ENFORCEMENT: FeatureFlagFromString,
```
[the remark this answers is not in the document]
Comment (salvatorelaiso) [reply utils/config.ts:131]: Solved in 2c19952

**Pull request merged**